providers/implementations/kem/ecx_kem.c.in

```diff
@@ -143,7 +143,7 @@ static ECX_KEY *ecxkey_pubfromdata(PROV_ECX_CTX *ctx,
     ecx = ossl_ecx_key_new(ctx->libctx, ctx->recipient_key->type, 1, ctx->propq);
     if (ecx == NULL)
         return NULL;
-    if (ossl_ecx_key_fromdata(ecx, params, 0) <= 0) {
+    if (ossl_ecx_key_fromdata(ecx, params, NULL, 0) <= 0) {
```

Comment:
Should this function be rewritten to make it clear the params is just a single OSSL_PARAM? Otherwise this is confusing, although it will work.

Reply:
Agreed, a good suggestion.